### DISCUSSION TITLE
[auth] Invalid copilot token: missing token: 403

### DISCUSSION THREAD

**Discussion by zhangbaidu2022** [community/community#114685] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

I subscribed to Copilot but cannot use it. The log in VSCode indicates:
[auth] Invalid Copilot token: missing token: 403

**Comment by CafuChino** [comment]
I encountered a similar issue a few days ago. According to the response from the ticket, the cause of the issue was that Github "perceived" an excessive or automated usage behavior from my end, leading to a unilateral ban on Copilot privileges without any notification or warning, even though I may have paid for this service. However, when I requested evidence to substantiate this action, my request was denied. Instead, I was repeatedly threatened with a permanent ban on my Github account if they still "perceive" similar behavior from me.

Therefore, I would like to alert those who have encountered this problem (or have yet to encounter it) to the following points:

Github may unilaterally suspend your account at any time.
Despite subscribing and paying for a feature you need, Github can still unilaterally disable this feature when they deem necessary, without providing any notification, explanation, or compensation.
Your data on Github is not secure, as Github can permanently ban your account when they deem necessary, especially when you seek to challenge their actions, they respond with threats.
Given these circumstances, I would advise, if possible, to consider other code completion tools, backup your data as much as possible, and turn to open-source solutions to minimize reliance on Github.

After this incident, I have lost confidence in Github's service level and data security. This is because, in the same ticket, I was threatened twice with a permanent account ban, when all I wanted to know was what I had done to warrant a ban on my paid subscription feature. All they did was issue threats and direct me to read the user agreement.

**Comment by ayim** [comment]
@zhangbaidu2022 what is the GitHub username you're trying to use with Copilot
@Akash1134 what would you recommend here

**Reply by Akash1134** [reply]
Hi @zhangbaidu2022 , I checked your account activity and It appears that your access to Copilot was restricted by our automated abuse detection mechanisms. Our systems is constantly learning and we are constantly adjusting to ensure proper abuse detection.

To get this sorted, I'd recommend opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

**Reply by ayim** [reply]
Since this is abuse related and the only next step is to contact support, I'm going to lock this thread.
